src/app/features/courses/components/courses-item/courses-item.component.scss

```diff
@@ -31,8 +32,16 @@
         box-sizing: border-box;
     }
 
+    &__body.top-rated{
+        background-color: red;
```

Comment:
minor: If you use variables for colors, use in all app them

Reply:
fixed: moved colors to variables.